tests/spread/ci/task.yaml

```diff
@@ -0,0 +1,49 @@
+summary: Build an LXD spread test image with TPM software
+manual: true
+artifacts:
+    - lxd-spread-core20-img.tar.gz
+
+execute: |
+    INSTANCE_NAME=spread-test
+    IMAGE_NAME=lxd-spread-core20-img
+
+    # be in project folder
+    cd $PROJECT_PATH
```

Comment:
for a following pr I recomend to run shellcheck on this files. it is very easy reusing some tooling we have in snapd repo.